pkg/controller/argocd/argocd_controller_test.go

```diff
@@ -19,7 +19,7 @@ import (
 )
 
 const (
-	argocdInstanceName = "argocd"
+	argocdInstanceName = "openshift-gitops"
```

Comment:
Instead of redefining the const in multiple files, we could use a common package to declare constants and reuse them everywhere, WDYT?  It'll be a single word change in the future

Something like https://github.com/argoproj/argo-cd/blob/master/common/common.go